src/language-markdown/parser-markdown.js

```diff
@@ -68,9 +68,10 @@ function htmlToJsx() {
       }
 
       return nodes.reduce((newNodes, { sourceSpan: position, type }) => {
-        const value = node.value
-          .slice(position.start.offset, position.end.offset)
-          .trim();
+        const value = node.value.slice(
+          position.start.offset,
+          position.end.offset
+        );
```

Comment:
It seems you forgot to revert this change.

Reply:
`value` should not be trimed.